src/BlockEntities/JukeboxEntity.cpp

```diff
@@ -24,19 +24,6 @@ cJukeboxEntity::cJukeboxEntity(BLOCKTYPE a_BlockType, NIBBLETYPE a_BlockMeta, Ve
 
 
 
-cJukeboxEntity::~cJukeboxEntity()
```

Comment:
IIRC, in the past, `Destroy` wasn't called for block entities when doing WE actions, hence the reason this piece of code existed. Seems like this behavior was changed/fixed at some point though, so this code can be deleted.

Reply:
You're right, thanks for raising this.